.github/workflows/ci.yml

```diff
@@ -31,6 +31,8 @@ env:
   # Minimum required Java version for running Ozone is defined in pom.xml (javac.version).
   TEST_JAVA_VERSION: 17 # JDK version used by CI build and tests; should match the JDK version in apache/ozone-runner image
   MAVEN_OPTS: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.class=standard -Dmaven.wagon.http.retryHandler.count=3
+  OZONE_RUNNER_IMAGE: ghcr.io/apache/ozone-runner
+  OZONE_RUNNER_VERSION: 20241108-jdk17-1
```

Comment:
Each directory in the compose folder has as .env file where these env variables are set differently.  Will this not override the value set in github CI job
```
OZONE_RUNNER_VERSION=${docker.ozone-runner.version}
OZONE_RUNNER_IMAGE=apache/ozone-runner
```

Reply:
No, `.env` only defines default values, which can be overridden in your shell.

Reply:
Try:

```
cd hadoop-ozone/dist/target/ozone-2.0.0-SNAPSHOT/compose/ozone
OZONE_RUNNER_IMAGE=bash OZONE_RUNNER_VERSION=5.1 ./test.sh
```

Reply:
Got it